modular_citadel/code/modules/client/loadout/suit.dm

```diff
@@ -69,6 +69,16 @@
 	category = SLOT_WEAR_SUIT
 	path = /obj/item/clothing/suit/hooded/wintercoat/aformal
 
+/datum/gear/coat/runed
+	name = "Runed winter coat"
+	category = SLOT_WEAR_SUIT
+	path = /obj/item/clothing/suit/hooded/wintercoat/narsie/fake
+
+/datum/gear/coat/brass
+	name = "Brass winter coat"
+	category = SLOT_WEAR_SUIT
+	path = /obj/item/clothing/suit/hooded/wintercoat/ratvar/fake
+
```

Comment:
I'm fairly against loadouts containing items that could be easily mistaken for antag gear.
But I'm not against the arcade prizes pool containing such trickeries. :clown_face:

Reply:
seems fair, I'll do that

Reply:
actually nah, it'll just be like the tactitool turtleneck, if that's fine?

Reply:
Ok fine. :/